### DISCUSSION TITLE
Errors Every Few Days

### DISCUSSION THREAD

**Discussion by samtietze** [mbround18/valheim-docker#632] [Q&A] [answered]
Hello all,

My friends message me practically every other day to let me know our server seems to have gone down, but I'm not sure how to change our configuration based on these errors. If it helps, we don't use mods or otherwise change anything beyond some typical settings in our docker file (e.g. what time to backup or update, etc.). I have seen some other errors at times, but this PlayFab thing seems to happen pretty frequently. I believe I last cloned this repo and set up the container at the beginning of November (and we started with a new world).
The computer running Docker is hardwired to our router, so I don't believe typical packet loss or weak signal is to blame.

Anybody have any ideas? I haven't seen any other threads about 'PlayFab' in the search, so maybe that's a red herring.

Edit: FWIW the issue is always resolved by stopping the container and then re-running `docker compose up`.

```
valheim-docker-valheim-1  | 12/08/2022 11:18:20: PlayFab reconnect server '[server name]'
valheim-docker-valheim-1  | 12/08/2022 11:18:20: Server '[server name]' begin PlayFab create and join network for server 
valheim-docker-valheim-1  | PARTY_STATE_CHANGE_RESULT_INTERNET_CONNECTIVITY_ERROR
valheim-docker-valheim-1  | the ping to the quality of service beacon timed out
valheim-docker-valheim-1  | PARTY_STATE_CHANGE_RESULT_INTERNET_CONNECTIVITY_ERROR
valheim-docker-valheim-1  | the region list could not be determined
valheim-docker-valheim-1  | the network operation cannot complete because creating the network failed
valheim-docker-valheim-1  | PARTY_STATE_CHANGE_RESULT_NETWORK_NO_LONGER_EXISTS
valheim-docker-valheim-1  | the network operation cannot complete because creating the network failed
valheim-docker-valheim-1  | Failed to send session data. Error: /Event/WriteTelemetryEvents: The requesting entity is not authorized to write events with namespace playfab.party. Allowed namespaces can be 'custom' or begin with 'custom.'.
```

**Comment by mbround18** [comment, marked as answer]
@samtietze this is an error directly with Valheim and not something i have control over :/ , you will have to ask them about this issue https://www.reddit.com/r/valheim/
